tests/common/plugins/sanity_check/__init__.py

```diff
@@ -90,6 +90,10 @@ def print_logs(duthosts, print_dual_tor_logs=False):
             cmds.remove(constants.PRINT_LOGS['mux_status'])
             cmds.remove(constants.PRINT_LOGS['mux_config'])
 
+        # check PTF device reachability
+        cmds.append("ping {} -c 1 -W 3".format(ptfhost.mgmt_ip))
```

Comment:
>mgmt_ip [](http://example.com/codeflow?start=55&length=7)

will it work if mgmt only has ipv6 address?

Reply:
Fixed, check both IPV4 and IPV6